src/languages/armasm.js

```diff
@@ -87,8 +96,8 @@ export default function(hljs) {
       {
         className: 'symbol',
         variants: [
+            {begin: '^[ ]*[a-z_\\.\\$][a-z0-9_\\.\\$]+:'}, //GNU ARM syntax
```

Comment:
The same question about white-spaces here.

Reply:
Same answer.